lib/Core/Memory.cpp

```diff
@@ -55,10 +111,10 @@ void MemoryObject::getAllocInfo(std::string &result) const {
 
   if (allocSite) {
     info << " allocated at ";
-    if (const Instruction *i = dyn_cast<Instruction>(allocSite)) {
+    if (const auto *i = dyn_cast<llvm::Instruction>(allocSite)) {
```

Comment:
Changes like this have little value imho, the existing code works fine, this doesn't change the semantics. All it does is make the history less clear, in turn making `git blame` harder and makes the diffs bigger